### PR TITLE
Implement imei validation

### LINKED ISSUE #539: Create IMEI validator
Create a new validator to check valid IMEIs (International Mobile Station Equipment Identity).

The rule is explained on this wiki https://en.wikipedia.org/wiki/International_Mobile_Station_Equipment_Identity

### PR DESCRIPTION
**Pull request by diegoholiveira**
Closes #539 

A little contrib for this awesome library :)

### CODE REVIEW

File: tests/integration/rules/imei_1.phpt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+require 'vendor/autoload.php';
+
+use Respect\Validation\Rules\Imei;
+
+$imei = new Imei();
+
+var_dump(
+    $imei->validate('490154203237518'),
+    $imei->validate('490154203237512'),
+    $imei->assert('490154203237518'),
```
Comment (batusa) [inline tests/integration/rules/imei_1.phpt]: would be nice if you check for true using the `check` method here

**Comment by batusa** [comment]
I think that you'll need to create a unit test class for your rule...

guidelines are here:
https://github.com/Respect/Validation/blob/master/CONTRIBUTING.md

**Comment by hussani** [comment]
Nice bro! 
:metal:

**Comment by henriquemoody** [comment]
Merged manually by 70997912eb748120c77cb584f7f761c99a3ea42d with some additions and changes.

@diegoholiveira, thanks for the pull request and thanks to @batusa for the core review! :panda_face:

**Pull request closed without merging**